modules/microshift-config-cli-manifests.adoc

```diff
@@ -0,0 +1,154 @@
+// Module included in the following assemblies:
+//
+// * microshift/using-config-tools.adoc
+
+:_content-type: CONCEPT
+[id="microshift-config-cli-manifests_{context}"]
+= Using CLI tools and creating manifests
+
+Configure your {product-title} using the supported command line arguments and environment variables.
+
+== Supported command-line arguments and environment variables
+
+[cols="4",options="header"]
+|===
+|Field Name
+|CLI Argument
+|Environment Variable
+|Description
+
+|clusterCIDR
+|--cluster-cidr
+|MICROSHIFT_CLUSTER_CLUSTERCIDR
+|A block of IP addresses from which Pod IP addresses are allocated
+
+|serviceCIDR
+|--service-cidr
+|MICROSHIFT_CLUSTER_SERVICECIDR
+|A block of virtual IP addresses for Kubernetes services
+
+|serviceNodePortRange
+|--service-node-port-range
+|MICROSHIFT_CLUSTER_SERVICENODEPORTRANGE
+|The port range allowed for Kubernetes services of type NodePort
+
+|dns
+|--cluster-dns
+|MICROSHIFT_CLUSTER_DNS
+|The Kubernetes service IP address where pods query for name resolution
+
+|domain
+|--cluster-domain
+|MICROSHIFT_CLUSTER_DOMAIN
+|Base DNS domain used to construct fully qualified pod and service domain names
+
+|url
+|--url
+|MICROSHIFT_CLUSTER_URL
+|URL of the API server for the cluster
+
+|nodeIP
+|--node-ip
+|MICROSHIFT_NODEIP
+|The IP address of the node, defaults to IP of the default route
+
+|nodeName
+|--node-name
+|MICROSHIFT_NODENAME
+|The name of the node, defaults to hostname
+
+|logVLevel
+|--v
+|MICROSHIFT_LOGVLEVEL
+|Log verbosity (0-5)
+|===
+
+== Auto-applying Manifests
+{product-title} uses `kustomize` for Kubernetes-native templating and declarative management of resource objects. Upon start-up, it searches `/etc/microshift/manifests` and `/usr/lib/microshift/` manifests directories for a `kustomization.yaml` file. If it finds one, it automatically runs the `kubectl apply -k command` to apply that manifest.
```

Comment:
I don't think we're actually using kustomize, yet. Are we @fzdarsky ?

Reply:
Left this module in until we are sure. I can comment it out if we're not using kustomize.